estate/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+{
+    'name': "Real Estate",
+
+    'summary': """
+        Cool Real Estate App
+    """,
```

Comment:
For multiple description/summary strings, there should be no indentation. See example [here](https://github.com/odoo-dev/odoo/blob/dbfd55ff6c86eb7dfc581ebaffd4c6a78972580f/addons/base_sparse_field/__manifest__.py#L4-L10)
```suggestion
    'summary': "Cool Real Estate App",
```